README.md

```diff
@@ -297,27 +297,27 @@ The OpenTelemetry Operator *might* work on versions outside of the given range,
-| v0.49.0                | v1.19 to v1.23       | 1.6.1                |
-| v0.48.0                | v1.19 to v1.23       | 1.6.1                |
-| v0.47.0                | v1.19 to v1.23       | 1.6.1                |
-| v0.46.0                | v1.19 to v1.23       | 1.6.1                |
-| v0.45.0                | v1.21 to v1.23       | 1.6.1                |
-| v0.44.0                | v1.21 to v1.23       | 1.6.1                |
-| v0.43.0                | v1.21 to v1.23       | 1.6.1                |
-| v0.42.0                | v1.21 to v1.23       | 1.6.1                |
-| v0.41.1                | v1.21 to v1.23       | 1.6.1                |
-| v0.41.0                | v1.20 to v1.22       | 1.6.1                |
-| v0.40.0                | v1.20 to v1.22       | 1.6.1                |
-| v0.39.0                | v1.20 to v1.22       | 1.6.1                |
-| v0.38.0                | v1.20 to v1.22       | 1.6.1                |
-| v0.37.1                | v1.20 to v1.22       | v1.4.0 to v1.6.1     |
-| v0.37.0                | v1.20 to v1.22       | v1.4.0 to v1.5.4     |
-| v0.36.0                | v1.20 to v1.22       | v1.4.0 to v1.5.4     |
-| v0.35.0                | v1.20 to v1.22       | v1.4.0 to v1.5.4     |
-| v0.34.0                | v1.20 to v1.22       | v1.4.0 to v1.5.4     |
-| v0.33.0                | v1.20 to v1.22       | v1.4.0 to v1.5.4     |
+| v0.51.0                | v1.19 to v1.23       | v1alpha2             |
+| v0.50.0                | v1.19 to v1.23       | v1alpha2             |
+| v0.49.0                | v1.19 to v1.23       | v1alpha2             |
+| v0.48.0                | v1.19 to v1.23       | v1alpha2             |
+| v0.47.0                | v1.19 to v1.23       | v1alpha2             |
+| v0.46.0                | v1.19 to v1.23       | v1alpha2             |
+| v0.45.0                | v1.21 to v1.23       | v1alpha2             |
+| v0.44.0                | v1.21 to v1.23       | v1alpha2             |
+| v0.43.0                | v1.21 to v1.23       | v1alpha2             |
+| v0.42.0                | v1.21 to v1.23       | v1alpha2             |
+| v0.41.1                | v1.21 to v1.23       | v1alpha2             |
+| v0.41.0                | v1.20 to v1.22       | v1alpha2             |
+| v0.40.0                | v1.20 to v1.22       | v1alpha2             |
+| v0.39.0                | v1.20 to v1.22       | v1alpha2             |
+| v0.38.0                | v1.20 to v1.22       | v1alpha2             |
+| v0.37.1                | v1.20 to v1.22       | v1alpha2, v1alpha3   |
+| v0.37.0                | v1.20 to v1.22       | v1alpha2, v1alpha3   |
+| v0.36.0                | v1.20 to v1.22       | v1alpha2, v1alpha3   |
+| v0.35.0                | v1.20 to v1.22       | v1alpha2, v1alpha3   |
+| v0.34.0                | v1.20 to v1.22       | v1alpha2, v1alpha3   |
+| v0.33.0                | v1.20 to v1.22       | v1alpha2, v1alpha3   |
```

Comment:
did we actually tested 0.33.0 against e.g. the oldest version of cert-manager v1alpha3?